src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -667,7 +667,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
         args_schema.identity = AAZResourceIdArg(
             options=["--identity"],
             help="Name or ID of the ManagedIdentity Resource.",
-            required=True,
```

Comment:
Why you made a change in network?

Reply:
Good catch. Not sure how that happened.  This should not be there, reverting.